stream/src/public_key/handshake.rs

```diff
@@ -26,12 +26,12 @@ pub struct Info<C: Scheme> {
 impl<C: Scheme> Info<C> {
     pub fn new(
         recipient: C::PublicKey,
-        secret: &x25519_dalek::EphemeralSecret,
+        ephemeral_public_key: x25519::PublicKey,
```

Comment:
Since we never actually use the secret in `Info`, I think it is clearer to just accept the ephemeral public key (we already use `x25519` directly in the connection, so calling the conversion function there isn't breaking some abstraction boundary we've tried to maintain)./